### PR TITLE
Update linkedin-learning.md

### PR DESCRIPTION
**Pull request by eoverfield**
Removed double use of free access

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/learning/linkedin-learning.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ description: Learn how to manage and use LinkedIn Learning in Viva Learning
 Select free content from LinkedIn Learning is available for everyone in Viva Learning. You can enable or disable LinkedIn Learning as a content source from the Microsoft 365 admin center.
 
 >[!NOTE]
```
Comment (JohanFreelancer9) [inline microsoft-365/learning/linkedin-learning.md:25]:
```suggestion
> [!NOTE]
```

File: microsoft-365/learning/linkedin-learning.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ description: Learn how to manage and use LinkedIn Learning in Viva Learning
...
->Note: Everyone has access free access to the select 125 free courses. For more courses, you'll need an enterprise subscription to LinkedIn Learning.
+>Note: Everyone has access to the select 125 free courses. For more courses, you'll need an enterprise subscription to LinkedIn Learning.
```
Comment (JohanFreelancer9) [inline microsoft-365/learning/linkedin-learning.md:26]:
```suggestion
>Note: Everyone has access to the select 125 free courses. For more courses, you'll need an enterprise subscription to LinkedIn Learning.
```
```suggestion
> Everyone has access to the select 125 free courses. For more courses, you'll need an enterprise subscription to LinkedIn Learning.
```

Comment (JohanFreelancer9) [inline microsoft-365/learning/linkedin-learning.md:26]:
```suggestion
> Everyone has access to the select 125 free courses. For more courses, you'll need an enterprise subscription to LinkedIn Learning.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @eoverfield, kindly commit the suggested additional changes. Many thanks.

**Comment by daisyfell** [comment]
@eoverfield and @JohanFreelancer9 thanks for your help on this article! Unfortunately, this article has since been retired.

**Pull request closed without merging**